src/internal_rules.jl

```diff
@@ -447,6 +447,99 @@ function EnzymeRules.reverse(config, func::Const{typeof(\)}, ::Type{RT}, cache,
     return (nothing,nothing)
 end
 
+
+function EnzymeRules.augmented_primal(
+    config,
+    func::Const{typeof(\)},
+    ::Type{RT},
+    A::Annotation{AT},
+    b::Annotation{BT}
+) where {RT, AT <: Union{UpperTriangular, LowerTriangular}, BT <: Array}
```

Comment:
Can the same rule also support `UnitUpperTriangular` and `UnitLowerTriangular`?

Reply:
Yes, done!